tests/all/integration/api/attendee/test_attendee_access_api.py

```diff
@@ -18,42 +18,6 @@ def get_minimal_attendee(db, user=None, owner=False, event_status='published'):
     return attendee
 
 
-def test_get_attendees_error(db, client, user, jwt, admin_user, admin_jwt):
```

Comment:
can i understand why get attendees test is removed? this is not post

Reply:
my bad, just updated the PR, it should be return 404.